mockery/generator.go

```diff
@@ -631,6 +631,8 @@ func (g *Generator) Write(w io.Writer) error {
 
 	res, err := imports.Process("mock.go", theBytes, opt)
 	if err != nil {
+		line := "--------------------------------------------------------------------------------------------"
+		fmt.Fprintf(os.Stderr, "Between the lines is the file (mock.go) mockery generated in-memory but detected as invalid:\n%s\n%s\n%s\n", line, g.buf.String(), line)
```

Comment:
Yep. I was addressing your main concern. I also mentioned what "additional information" I wanted to convey:

> I didn't know what mock.go was until looking in the source. The errors that I've encountered at that site have all mentioned mock.go.

The message explains [what mock.go is](https://github.com/vektra/mockery/issues/119#issue-207597729). We agree it doesn't help with debugging (future work). Instead, it just clarifies what just went wrong (past event).

Put another why: it explains why mockery just stopped and didn't do the work that was expected.

Is that TMI?